src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -5,17 +5,21 @@ import {
   isCurrentVenueNGRequestingSelector,
 } from "utils/selectors";
 
-import { IFRAME_TEMPLATES } from "settings";
-
 import { useSelector } from "hooks/useSelector";
 import { useIsUserVenueOwner } from "hooks/useIsUserVenueOwner";
 import { useUser } from "hooks/useUser";
 import { useVenueId } from "hooks/useVenueId";
 import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useShowHide } from "hooks/useShowHide";
 
 import { LoadingPage } from "components/molecules/LoadingPage";
-import { IframeAdmin } from "components/molecules/IframeAdmin";
+
 import { BannerAdmin } from "components/organisms/BannerAdmin";
+import { WithNavigationBar } from "components/organisms/WithNavigationBar";
+
+import { AnnouncementMessage } from "components/molecules/AnnouncementMessage";
```

Comment:
Align with the other molecule (LoadingPage)

Reply:
fixed 2431677